radixdlt/src/test/java/com/radixdlt/sanitytestsuite/scenario/SanityTestScenarioRunner.java

```diff
@@ -33,12 +32,9 @@
 
 public abstract class SanityTestScenarioRunner<TestVector extends SanityTestVector> {
 
-	private static final Logger log = LogManager.getLogger();
-
 	public abstract String testScenarioIdentifier();
 
-	public abstract TypeToken<TestVector> typeOfVector();
-
+	public abstract TypeReference<TestVector> testVectorTypeReference();
```

Comment:
Would be nice to be able to skip the `TypeReference<TestVector> testVectorTypeReference();` solution all together. But challenging because Java typeerases the Generic type. How to do it? See comment below regarding casting of `UnknownTestVector`.